src/components/MessageHTMLBody.vue

```diff
@@ -3,9 +3,16 @@
 		<MdnRequest :message="message" />
 		<div v-if="hasBlockedContent" id="mail-message-has-blocked-content" style="color: #000000">
 			{{ t('mail', 'The images have been blocked to protect your privacy.') }}
-			<button @click="onShowBlockedContent">
-				{{ t('mail', 'Show images from this sender') }}
-			</button>
+			<Actions default-icon="icon-toggle">
+				<ActionButton icon="icon-toggle"
+					@click="onShowBlockedContent">
+					{{ t('mail', 'Show images from {sender}', {sender: message.from[0].email}) }}
+				</ActionButton>
+				<ActionButton icon="icon-toggle"
+					@click="onShowBlockedDomain">
+					{{ t('mail', 'Show images from this {domain}', {domain: message.from[0].email.split('@').pop()}) }}
```

Comment:
can we move the email to domain conversion into a method and use it here and in the other method below?